app/views/internal/users/_mentee_match.erb

```diff
@@ -0,0 +1,66 @@
+<% if @user.seeking_mentorship && @user.mentee_description.present? && !@user.banned_from_mentorship  %>
+  <h2> Mentee Description </h2>
+    <div class="row" style="font-size:0.85em;position:sticky;top:50px;background:white;z-index:1000;border:2px solid black">
+      <div class="col-md-2">
+        <img src="<%= @user.profile_image_90 %>" style="border-radius:100px;width:50px;height:50px;" />
+        <br/>
+        <a href="/<%= @user.username %>" target="_blank">@<%= @user.username %></a>
+        <br/>
+      </div>
+      <div class="col-md-3" style="border-left:1px solid black">
+        <h4>Tags</h4>
+        <%= @user.cached_followed_tag_names.join(", ") %>
+      </div>
+      <div class="col-md-3" style="max-height: 22vh;overflow:auto">
+        <h4>Mentee description</h4>
+        <%= @user.mentee_description %>
+      </div>
+      <div class="col-md-3">
+        <h4>Skills/Languages</h4>
+        <%= @user.mostly_work_with %>
+      </div>
+      <div class="col-md-1">
+      </div>
+    </div>
+
+  <h2> Possible Mentors </h2>
+  <% count = 0 %>
+  <% MentorRelationship.
```

Comment:
Hmm, the method chain here is bewildering! Is it possible to stuff all these away into either a presenter or create a scope in the model?

Reply:
Yeah, @jessleenyc I think we can plop more stuff like this into `article_decorator` to clean things up and for re-use in the future.